### PR TITLE
AT 05.02 | Multibranch Pipeline > Rename Multibranch Pipeline

### PR DESCRIPTION
**Pull request by arturpuchkof**
AT 05.02 | Multibranch Pipeline > Rename Multibranch Pipeline
https://trello.com/c/eFCjldAX

### CODE REVIEW

**Comment by arturpuchkof** [comment]
[AT 05.02 | Multibranch Pipeline > Rename Multibranch Pipeline](https://trello.com/c/eFCjldAX/1374-at-0502-multibranch-pipeline-rename-multibranch-pipeline)

**Review by ElNov686 — APPROVED** [review]

File: src/test/java/school/redrover/RenameMultibranchPipelineTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+import school.redrover.runner.TestUtils;
+
+public class RenameMultibranchPipelineTest extends BaseTest {
+
+    @Test
+    public void testRenameMultibranchPipeline(){
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/RenameMultibranchPipelineTest.java:15]: Please change the name of class like "MultibranchPipeline4Test"

Comment (arturpuchkof) [reply src/test/java/school/redrover/RenameMultibranchPipelineTest.java:15]: Why?

Comment (arturpuchkof) [reply src/test/java/school/redrover/RenameMultibranchPipelineTest.java]: fixed

File: src/test/java/school/redrover/RenameMultibranchPipelineTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        renameButton.click();
+
+        WebElement nameInputField = getDriver().findElement(By.name("newName"));
+        nameInputField.clear();
+        nameInputField.sendKeys(newName);
+        nameInputField.sendKeys(Keys.ENTER);
+
+        WebElement getTitle = getDriver().findElement(By.xpath("//h1"));
+
+        Assert.assertEquals(newName, getTitle.getText());
```
[the remark this answers is not in the document]
Comment (arturpuchkof) [reply src/test/java/school/redrover/RenameMultibranchPipelineTest.java]: fixed

**Review by nikabenz — APPROVED** [review]

**Review by evanmaiqa — APPROVED** [review]

File: src/test/java/school/redrover/RenameMultibranchPipelineTest.java (hunk 1 above)
Comment (evanmaiqa) [reply src/test/java/school/redrover/RenameMultibranchPipelineTest.java]: We are working with the "MultibranchPipelineTest" class and inside the class we can rename, change, configure, modify, delete. We don't need to create a class for every action.

Comment (arturpuchkof) [reply src/test/java/school/redrover/RenameMultibranchPipelineTest.java]: who are we?
your particular group?

**Pull request merged**